bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -103,6 +123,10 @@ spec:
           volumeMounts:
             - name: data
               mountPath: {{ .Values.persistence.mountPath }}
+            {{- if .Values.persistence.commitStorageClass }}
```

Comment:
as the `storageClass` is not mandatory, it is better to use them `commitLogMountPath` because otherwise if you run:
```
helm template cassandra cassandra/ --set persistence.commitStorageClass='-' --set persistence.commitLogsize='2Gi' --set persistence.commitLogMountPath='/bitnami/cassandra/commitlog'
```
at the end of the file appear this line
```
 storageClassName: ""
```
And the pvc `commitlog-cassandra-0` never get bound

Could you change this for `{{- if .Values.persistence.commitLogMountPath }}`?